test/format/config.c

```diff
@@ -252,6 +246,56 @@ config_setup(void)
 	g.key_cnt = 0;
 }
 
+/*
+ * config_cache --
+ *	Cache configuration.
+ */
+static void
+config_cache(void)
+{
+	uint32_t required;
+
+	if (!config_is_perm("cache")) {
+		/* Ensure there is at least 1MB of cache per thread. */
+		if (g.c_cache < g.c_threads)
+			g.c_cache = g.c_threads;
+
+		/* Check if a minimum cache size has been specified. */
+		if (g.c_cache_minimum != 0 && g.c_cache < g.c_cache_minimum)
+			g.c_cache = g.c_cache_minimum;
+	}
+
+	/*
+	 * Maximum internal/leaf page size sanity.
+	 * Ensure we can service at least one operation per-thread concurrently
+	 * without filling the cache with pinned pages. We choose a multiplier
+	 * of three because the max configurations control on disk size and in
```

Comment:
I realize that this comment and code is moved from another location and this isn't freshly written, but I'd remove "of three" from the comment because it doesn't actually say why 3 itself is a good number and not, say, 4. Since it is used in several places, can you make it a macro? `#define CACHE_MULTIPLIER`. It also makes me sad that we have the dirty trigger value hard coded. Can that be a macro?

Reply:
Pushed in 632b009.